leakcanary-android-instrumentation/src/androidTest/java/com/squareup/leakcanary/InstrumentationTestApplication.java

```diff
@@ -5,9 +5,6 @@
 public class InstrumentationTestApplication extends Application {
   @Override public void onCreate() {
     super.onCreate();
-    if (LeakCanary.isInAnalyzerProcess(this)) {
-      return;
-    }
```

Comment:
Note: somewhat unrelated but I thought of this when writing the fragment tests. We're never running the leak canary process during instrumentation tests, so no need for this.